src/utils/parseRelativeTime.ts

```diff
@@ -51,3 +53,36 @@ export const parseRelativeTime = (target: Date): string => {
 
   return `${diffInYears}年前`;
 };
+
+/**
+ * 「今日 10:00」や「2023/01/01 10:00」のような形式に変換する
+ * @param target 日付
+ * @returns 
+ */
+export const parseFixedDateTime = (target: Date): string => {
```

Comment:
日付の判定ロジックを自前で実装しなくても、date-fns-tzが提供してくれているようです。
このようにしてみてはどうでしょうか。
変数を書き換えずに、よりロジックを簡潔に実装できます。

```suggestion
export const parseFixedDateTime = (target: Date): string => {
  const timeZone = "Asia/Tokyo";
  target = utcToZonedTime(target, timeZone);

  const formattedTime = format(target, "HH:mm");

  if (isToday(target)) {
    return `今日 ${formattedTime}`;
  }

  if (isYesterday(target)) {
    return `昨日 ${formattedTime}`;
  }

  return `${format(target, "yyyy/MM/dd")} ${formattedTime}`;
};

```